app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -0,0 +1,250 @@
+        )
+      ]
+    */
+
+    // Fetch data
+    val dbIO = for {
+      exportDefinition <- StacExportDao.unsafeGetById(params.stacExportId)
+      _ <- setExportStatus(exportDefinition, ExportStatus.Exporting)
+      layerSceneTaskAnnotation <- sceneTaskAnnotationforLayers(
+        exportDefinition.layerDefinitions,
+        exportDefinition.taskStatuses
+      )
+    } yield (exportDefinition, layerSceneTaskAnnotation)
+    val contentBundle =
+      (ContentBundle.apply _).tupled(dbIO.transact(xa).unsafeRunSync)
+
+    // Construct STAC datamodel
+    /*
+     Exported Catalog:
+     |-> Layer collection
+     |   |-> Scene Collection
+     |   |   |-> Scene Item
+     |   |   |-> Scene Item
+     |   |   |-> (One or more scene items)
+     |   |-> Label Collection
+     |   |   |-> Label Item (Only one)
+     |   |   |-> Label Data in GeoJSON Feature Collection
+     |-> (One or more Layer Collections)
+
+     Final structure is going to be on s3
+    */
+    val catalogBuilder =
+      new StacCatalogBuilder[StacCatalogBuilder.CatalogBuilder.EmptyCatalog]()
+    val stacVersion = "0.7.0"
+    val currentPath =
+      contentBundle.export.exportLocation
+        .getOrElse("s3://rasterfoundry-production-data-us-east-1/stac-exports")
+    val catalogId = contentBundle.export.id.toString
+    val catalogParentPath = s"${currentPath}/${catalogId}"
+    val catalogDescription =
+      s"Exported from Raster Foundry ${(new Timestamp((new Date()).getTime())).toString()}"
+    val catalogOwnLinks = List(
+      StacLink(
+        // s3://rasterfoundry-production-data-us-east-1/stac-exports/<catalogId>/catalog.json
+        s"${catalogParentPath}/catalog.json",
+        Self,
+        Some(`application/json`),
+        Some(s"Catalog ${catalogId}")
+      ),
+      // s3://rasterfoundry-production-data-us-east-1/stac-exports/<catalogId>/catalog.json
+      StacLink(
+        s"${catalogParentPath}/catalog.json",
+        StacRoot,
+        Some(`application/json`),
+        Some(s"Catalog ${catalogId}")
+      )
+    )
+    val (
+      catalog,
+      layerSceneLabelCollectionsItemsAssets
+    ): (
+        StacCatalog, // catalog
+        List[
+          (
+              StacCollection, // layer collection
+              (StacCollection, List[StacItem]), // scene collection and scene items
+              (StacCollection, StacItem, (Option[Json], String)) // label collection, label item, label data, and s3 location
+          )
+        ]
+    ) = catalogBuilder
+      .withVersion(stacVersion)
+      .withParentPath(catalogParentPath, true)
+      .withId(contentBundle.export.id.toString)
+      .withTitle(contentBundle.export.name)
+      .withDescription(catalogDescription)
+      .withLinks(catalogOwnLinks)
+      .withContents(contentBundle)
+      .build()
+
+    // Write STAC catalog etc to s3
+    writeToS3(catalog, layerSceneLabelCollectionsItemsAssets)
```

Comment:
I'd comment this out and leave a TODO for the issue that will be doing this so that the job at least runs